### ISSUE TITLE
About the Models

### ISSUE THREAD

**Issue by Adeel-Intizar** [manhminno/Face-Recognition#1]
Hello, Thank you for your great work
can you tell me which Model is used for which Purpose?

classify_model.h5
facenet_keras.h5

Thank you

**Comment by manhminno** [comment]
Hi @Adeel-Intizar ,
- classify_model.h5 is weight of softmax model to classify vector which is embedded by facenet.
- facenet_keras.h5 is weight of facenet (model for embedding faces into 128 dimensional vectors - trained by MS-Celeb-1M dataset)

**Comment by Adeel-Intizar** [comment]
Thank you
